src/popup/router/components/AccountSwitcher.vue

```diff
@@ -7,13 +7,21 @@
       :class="['cards-wrapper', { 'menu-under': filteredAccounts.length > 1 }]"
       :style="cssVars"
     >
-      <AccountCard
+      <div
         v-for="account in filteredAccounts"
         :key="account.address"
-        :class="{ selected: account.i === activeIdx }"
-        v-bind="account"
-        :account-idx="account.i"
-      />
+      >
+        <AddAccountCard
+          v-if="account.i === 'add-account'"
+          :class="{ selected: account.i === activeAccount }"
+        />
+        <AccountCard
+          v-else
+          :class="{ selected: account.i === activeAccount }"
+          v-bind="account"
+          :account-idx="account.i"
+        />
```

Comment:
Maybe we could shorten it like that?
```suggestion
        <component
          :is="account.i === 'add-account' ? 'AddAccountCard' : 'AccountCard'"
          :class="{ selected: account.i === activeAccount }"
          v-bind="account"
          :account-idx="account.i"
        />
```